### DISCUSSION TITLE
You are Offline!

### DISCUSSION THREAD

**Discussion by himanshusankhala04** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#310] [General]
I am connected to internet still getting this error

**Comment by im-coder-lg** [comment]
Can you show a screenshot, even settings, but you can cut out the authentication part.

**Comment by himanshusankhala04** [comment]
Issue is solved I have to create new token for it

**Reply by im-coder-lg** [reply]
Did it work?

**Reply by himanshusankhala04** [reply]
Yes
